docs/Containers/Container-Tasks.md

```diff
@@ -118,6 +118,52 @@ The following is the process by which you can create a usable image from your cu
     Evergreen YAML must also use a **&lt;SHA&gt;** tag, which would be the hash
     of the corresponding commit in our [image repository](https://github.com/evergreen-ci/container-initial-offering-dockerfiles).
 
+The following is a starter Dockerfile that abides by our image policy and
+includes commonly used tools and packages (e.g. Go, Python, NodeJS, etc.)
+that you can copy/paste and use as a reference for creating your own. 
+
+``` dockerfile
+# Use an approved base image
+FROM ubuntu:latest
+
+# Required label that points to an owning team, per our image policy
+LABEL mongodb.maintainer="Your Team Name <team-email@company.com>"
+
+# Update package inventory
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update
+
+# Common tools
+RUN apt-get install -y \
+    curl \
+    wget \
+    git
+
+# Install Go
+RUN wget https://dl.google.com/go/go1.17.2.linux-amd64.tar.gz -O go.tar.gz \
+    # Must verify the checksum of downloaded file
+    && md5sum go.tar.gz | cut -d ' ' -f 1 | grep -xq b7af894763e397335efe5a9ca70a5d63 \
+    && tar -C /usr/local -xzf go.tar.gz \
+    && rm go.tar.gz
+ENV PATH=$PATH:/usr/local/go/bin
+
+# Install Node.js and npm
+RUN curl -sL https://deb.nodesource.com/setup_16.x -o nodesource_setup.sh \
+    # Must verify the checksum of downloaded file
+    && md5sum nodesource_setup.sh | cut -d ' ' -f 1 | grep -xq 26ac33def0855ea2c8dddfe361e6d313 \
+    && bash nodesource_setup.sh \
+    && apt-get install -y nodejs
+
+# Install Python3 and pip
+RUN apt-get install -y \
+    python3 \
+    python3-pip
+
+# Default to bash shell, as required by the Evergreen agent
+CMD [ "bin/bash" ]
```

Comment:
Not sure if the absolute path is needed since it seems like the bash shell starts up properly regardless, but I changed it to absolute just in case.

Thanks for the tip - changed to entrypoint and confirmed that works.

Reply:
In general, if you know the location of the entrypoint is a fixed, static location and not actually relative to the current working directory, it's preferable to use the absolute path since the default working dir (which is `/`) can be modified by a [`WORKDIR` directive](https://docs.docker.com/engine/reference/builder/#workdir) if the user chooses to add one.

If the past is any indicator of why relative paths are hard, host provisioning used to have substantial issues with relative paths vs. absolute paths due the fact that provisioning commands depended heavily on what prior `cd`'s were already run leading up to a relative path command. Changing the relative paths to be absolute paths simplified things down a lot.